src/main/java/io/github/thebusybiscuit/slimefun4/utils/HeadTexture.java

```diff
@@ -62,6 +62,7 @@
     FUEL_BUCKET("a84ddca766725b8b97413f259c3f7668070f6ae55483a90c8e5525394f9c099"),
     ELECTRIC_PRESS("8d5cf92bc79ec19f4106441affff1406a1367010dcafb197dd94cfca1a6de0fc"),
     ENERGY_REGULATOR("d78f2b7e5e75639ea7fb796c35d364c4df28b4243e66b76277aadcd6261337"),
+    ENERGY_CONNECTOR("ewogICJ0aW1lc3RhbXAiIDogMTYwMjMzMjA1MzM4NywKICAicHJvZmlsZUlkIiA6ICIyODkzNGZhODc3Yzg0MGQ4YjdhNWZiMWZmZTBkMDJiNiIsCiAgInByb2ZpbGVOYW1lIiA6ICJhamFuXzEyIiwKICAidGV4dHVyZXMiIDogewogICAgIlNLSU4iIDogewogICAgICAidXJsIiA6ICJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzEwODVlMDk4NzU2Yjk5NWIwMDI0MTY0NDA4OWM1NWE4ZjlhY2RlMzViOWEzNzc4NWQ1ZTA1N2E5MjM2MTNiIgogICAgfQogIH0KfQ=="),
     NETHER_ICE("3ce2dad9baf7eaba7e80d4d0f9fac0aab01a76b12fb71c3d2af2a16fdd4c7383"),
     ENRICHED_NETHER_ICE("7c818aa13aabc7294838d21caac057e97bd8c89641a0c0f8a55442ff4e27"),
     NETHER_ICE_COOLANT_CELL("8d3cd412555f897016213e5d6c7431b448b9e5644e1b19ec51b5316f35840e0"),
```

Comment:
Spot the difference...

Reply:
Blame @J3fftw1 his tutorial gave this result.

Reply:
Dont tag me also you could have used the shorter hash

Reply:
You need the hex texture id though, not the base64 json properties.
Simply decode this base64 mess, copy the texture url and snip off the domain. Then you get the hex id.

For the future I would recommend you to just use a tool like https://minecraft-heads.com/custom-heads/heads-generator

Reply:
So I need a base64 of https://textures.minecraft.net/texture/1085e098756b995b00241644089c55a8f9acde35b9a37785d5e057a923613b ?

Reply:
No you just need the hex, the last part of this url...
10853...613b

Reply:
1085e098756b995b00241644089c55a8f9acde35b9a37785d5e057a923613b

Reply:
@J3fftw1 btw this is another reason to not use that tutorial.

Reply:
Okay idc if you cant use it and stop tagging me